files/requirements.py

```diff
@@ -107,6 +113,11 @@ def setup_python(python: Python, source_directory: str, final: bool) -> None:
     display.section(f'Checking pip integrity for Python {python.version}')
     pip.check()
 
+    if python.version == '3.12':
+        display.info(f'Skipping checking PyYAML for libyaml support for Python {python.version}')
```

Comment:
Skipping the `libyaml` check on Python 3.12 isn't needed. While it doesn't always work with new Python versions, there doesn't appear to be any issue with Python 3.12 at this time.